code/game/jobs/job/police.dm

```diff
@@ -140,8 +140,8 @@
 	title = "City Hall Guard"
 	email_domain = "secure.plux.gov.nt"
 	flag = BRIDGE
-	department = DEPT_COUNCIL
-	department_flag = ENGSEC
+	department = "Civilian"
```

Comment:
We're using the new department system now, don't use "Civilian" use DEPT_PUBLIC, DEPT_RESEARCH, or DEPT_POLICE, etc. See department_presets.dm for list of them.

Reply:
Roger.